k8s/service_controller/server/nil/storage.go

```diff
@@ -79,6 +79,10 @@ func (s *NilServiceStorage) GetBrokerByService(id string) (*model.ServiceBroker,
 	return nil, nil
 }
 
+func (s *NilServiceStorage) AllServices() ([]*model.Service, error) {
```

Comment:
that's why I'm a bit confused. It compiles for me w/o this method. I didn't see any ref to AllServices() in any interface.